config/locales/en.yml

```diff
@@ -572,6 +572,19 @@ en:
       applications:
         create:
           notice: Application Registered.
+  errors:
+    contact:
+      html: Feel free to <a href="http://wiki.openstreetmap.org/wiki/Contact" title="Various contact channels explained">contact</a> the OpenStreetMap community if you have found a broken link / bug. Make a note of the exact URL of your request.
+    forbidden:
+      title: Forbidden
+      paragraph_1: The operation you requested on the OpenStreetMap server is only available to administrators (HTTP 403)
+    internal_server_error:
+      title: Application error
+      paragraph_1: The OpenStreetMap server encountered an unexpected condition that prevented it from fulfilling the request (HTTP 500)
+      paragraph_2: This may be a problem in our Ruby On Rails code. 500 occurs with exceptions thrown outside of an action (like in Dispatcher setups or broken Ruby code)
```

Comment:
I think the second sentence here is jargon that we shouldn't show to end users. I mention it here because I don't know what "Dispatcher setups" are, in this context, and so I wouldn't be able to advise a translator what it means either! :smile:

Reply:
Agreed... It's not really helpful so I've removed it.